.idea/misc.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can you .gitignore this file? Somehow it always gets removed and recreated by goland :D

Reply:
yes! 👍